packages/doctor-sdk/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
   "extends": "@rsbuild/tsconfig/base",
   "compilerOptions": {
+    "experimentalDecorators": true,
+    "moduleResolution": "node",
```

Comment:
Why add moduleResolution? it is already defined in base config

Reply:
done